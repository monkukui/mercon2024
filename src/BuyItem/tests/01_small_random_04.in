5
171613360 915498654 2
685141068 589705003 64
367426593 374406059 38
756249961 631020384 26
696078366 698121309 56
