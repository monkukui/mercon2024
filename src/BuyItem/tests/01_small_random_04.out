5
374406059 367426593
589705003 685141068
631020384 756249961
698121309 696078366
915498654 171613360
